### ISSUE TITLE
Замена клиента в доставочном заказе

### ISSUE THREAD

**Issue by Keeplod** [iiko/front.api.sdk#346]
Можно ли реализовать замену клиента средствами iiko front sdk?
Используя методы RemoveOrderClient, AddClientToOrder ни к чему хорошему не привели.
RemoveOrderClient и ручная кнопка (скидки надбавки - удалить клиента) удаляют только визуально, по факту гость остается в заказе, почему так?

**Comment by vcpp** [comment]
Нет, клиент доставочного заказа указывается при создании и затем не может быть заменён. Расскажите, пожалуйста, в каком сценарии это может иметь смысл?

Сейчас единственный вариант замены — это отменить старый заказ и создать новый.

`AddClientToOrder` и `RemoveOrderClient` — это про привязку клиента к заказу с точки зрения системы лояльности. В отличие от клиента доставки, который [`IDeliveryOrder.Client`](https://iiko.github.io/front.api.sdk/v7/html/P_Resto_Front_Api_Data_Orders_IDeliveryOrder_Client.htm) и который строго один (либо `null` в анонимной доставке), привязанных к заказу клиентов, которые [`IOrder.CustomerIds`](https://iiko.github.io/front.api.sdk/v7/html/P_Resto_Front_Api_Data_Orders_IOrder_CustomerIds.htm), может быть сколько угодно.

**Comment by Keeplod** [comment]
Бываю моменты когда клиент хочет заменить номер телефона, указывая номер другого клиента.
